src/patternfly/components/Popover/examples/Popover.md

```diff
@@ -51,12 +63,18 @@ import './Popover.css'
 ```hbs isFullscreen
 {{#> popover popover--modifier="pf-m-bottom" popover--attribute='aria-labelledby="popover-bottom-header" aria-describedby="popover-bottom-body"'}}
   {{#> popover-content}}
+    {{#> popover-close}}
       {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Close"'}}
         <i class="fas fa-times" aria-hidden="true"></i>
       {{/button}}
-      {{#> title titleType="h1" title--modifier="pf-m-md" title--attribute='id="popover-bottom-header"'}}
-        Popover header
-      {{/title}}
+    {{/popover-close}}
+    {{#> popover-header}}
+      {{#> popover-title popover-title--attribute='id="popover-bottom-header"'}}
```

Comment:
@thatblindgeye wdyt about this `id` (used as the value for `aria-labelledby` on the outer popover element) - it used to be on the heading element, but this change is moving it to the "title" `<div>`. Is that ok? Or should it preferably be on the heading element? If the latter, I think the same isue about the sr-only text applies here, too. If we prefer the `id` go on the heading itself, and a user provides their own heading, it would be up to them to add the `id`.